app-text/podofo/podofo-0.10.1.ebuild

```diff
@@ -54,3 +54,15 @@ src_configure() {
 	)
 	cmake_src_configure
 }
+
+src_test() {
+	local CMAKE_SKIP_TESTS=(
+		$(usex XXX '' 'testCMYKConstructor')
```

Comment:
Sorry, I did not mean to commit this lol. This was part of my late night debug/rebuild/test cycle trying to figure out why this failed.

But well, it's also the line that needs a final resolution to decide what to do about it w.r.t. https://github.com/podofo/podofo/issues/6#issuecomment-1731237023 so, lets discuss that.

Reply:
@thesamesam helped me debug this, it turns out that -ffp-contract=off allows this test to pass.